app/src/lib/rules/syntax_rules.js

```diff
@@ -120,6 +120,17 @@ const builtin_syntax_rules = [
 			}),
 		},
 	},
+	{
+		name: 'Tag words with a possessive \'s as genitive_saxon',
```

Comment:
```suggestion
		name: "Tag words with a possessive 's as genitive_saxon",
```

if you like that a little better...